FirebaseVertexAI/CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
 # Unreleased
```

Comment:
Might as well mark it 11.4.0 for clarity now.

Reply:
Done.